### ISSUE TITLE
content-type not being set as per docs to "application/json"

### ISSUE THREAD

**Issue by mattrlong** [segmentio/analytics-node#275]
According to the Segment api [documentation]( https://segment.com/docs/connections/sources/catalog/libraries/server/http-api/#content-type) the content-type should be set to `'application/json'` however while using this library the content-type header is coming through as `text/plain`.

**Comment by pbassut** [comment]
@mattrlong Hey! On request headers or response headers.

If on request: axios uses [Content-Type': 'application/x-www-form-urlencoded](https://github.com/axios/axios/blob/d99d5faac29899eba68ce671e6b3cbc9832e9ad8/lib/defaults.js#L7) by default.

If on response headers, then it's the server you're making requests to that's returning text/plain as it's content-type.

**Comment by mattrlong** [comment]
Hi @pbassut apologies, I was referring to the request headers of the requests made to segment's API using this library. As per the docs referenced above this should be set (overriding Axios' default) to `application/json`. 

I have spotted this issue, as our project is using `analytics-node` to integrate with Segment, and our automated test tool is noticing the `text/plain` header and forcing us to parse the text to JSON. If the header were set correctly we'd immediately be able to interrogate the JSON to confirm we've set the properties correctly.
